Dockerfile.martin

```diff
@@ -1,3 +1,3 @@
-FROM urbica/martin
```

Comment:
Wo ist der Unterschied zwischen denn beiden Quellen?

Reply:
`urbica/martin` is das alte Image, die Dokumentation verweist jetzt auf `maplibre/martin`; Letzteres hat Pushes, die Ersteres nicht (mehr) hat.